src/content/components/PriorityGuide/PriorityGuide.scss

```diff
@@ -34,3 +33,57 @@
 .p5 {
   color: #7e7e7e;
 }
+
+.sidebarToggle {
+  position: absolute;
+  right: 10px;
+  border: 0;
+  background: none;
+  height: 20px;
+}
+
+.active {
+  background: #f9f9fa;
+  padding: 20px;
+  position: absolute;
+  width: 250px;
+  top: 0px;
+  right: 0;
+  bottom: 0;
+
+  -moz-transition: right 0.25s ease;
```

Comment:
no need for `moz-transition`! I hope no one is using <Firefox 16

Reply:
Also,  using `translateX` here is better for performance than using `right` (see this article for more if you're interested https://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/)

Reply:
Is that something you'd like me to change? Or just an FYI :)

Reply:
Please change yeah

Reply:
okay, will do!